assets/questions/reverseWords.js

```diff
@@ -303,8 +304,10 @@ globalData.questions['reverseWords'] = {  // eslint-disable-line dot-notation
       ignoredTestSuiteIds: [],
       messages: [
         [
-          "Try reading the question again, and looking at the sample ",
-          "input/output -- it's asking for something different."
+          "I think you might have misinterpreted the question. Try verifying ",
```

Comment:
I recommend removing "I think". I don't think it's necessary to use pronouns to refer to the app (I think we should rely on the intelligence of the feedback to make it appear more human-like).

Reply:
Done, thanks -- I was actually trying to make the feedback "softer", but I'm fine either way and have removed it for now.